lib/krane/cli/render_command.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module Krane
+  module CLI
+    class RenderCommand
+      OPTIONS = {
+        bindings: { type: :array, desc: 'Bindings for erb' },
+        f: { type: :array, required: true, desc: 'Directories and files to render' },
```

Comment:
should we support it with an alias of `:filenames` like we intend to do with `krane deploy`?